Moonlight Config Fixes

Joe-K2018 opened the pull request:
Added all available AV Decoder options available in Moonlight Embedded.

Fixed formatting issue in changelog from original PR.

nadenislamarre commented:
not correct
"DEFAULT":               0  line must be removed.
and
if system.config["moonlight_avdecoder"] == "sdl":

Joe-K2018 commented:
> not correct "DEFAULT": 0 line must be removed. and if system.config["moonlight_avdecoder"] == "sdl":

I have removed the default line in es_features.yml and added the sdl conditional in moonlightConfig.py
